src/sheldon_suggestions_server.erl

```diff
@@ -0,0 +1,70 @@
+-module(sheldon_suggestions_server).
+
+-behaviour(gen_server).
+
+%%% API
+-export([ suggestions/2
+        , add_suggestions/2
+        ]).
+
+%%% gen_server callbacks
+-export([
+  init/1,
+  handle_call/3,
+  handle_cast/2,
+  handle_info/2,
+  terminate/2,
+  code_change/3
+]).
+
+%%%=============================================================================
+%%% API
+%%%=============================================================================
+
+-spec suggestions([sheldon_result:misspelled_word()], sheldon_dictionary:language()) ->
+  [sheldon_result:misspelled_word()].
+suggestions(MisspelledWords, Lang) ->
+  try rpc:pmap({?MODULE, add_suggestions}, [Lang], MisspelledWords) of
+    Result -> Result
+  catch
+    exit:badrpc -> MisspelledWords
```

Comment:
You should log the error here. I recommend using `error_logger:warning_msg/2` and including the stack trace in the report